packages/sanity/src/_internal/cli/server/buildVendorDependencies.ts

```diff
@@ -97,7 +97,7 @@ const VENDOR_IMPORTS: VendorImports = {
   },
   'styled-components': {
     '^6.1.0': {
-      '.': './dist/styled-components.esm.js',
+      '.': './dist/styled-components.browser.esm.js',
```

Comment:
this one was the wrong entry point to begin with 😅

Reply:
Good catch!!